docs/src/tut/pos/index.md

```diff
@@ -0,0 +1,243 @@
+# {#point-of-sale} Point-of-sale
+
+This tutorial will walk you through creating a business point-of-sale machine, like Toast or Square.
+
+It assumes that you are comfortable with writing simple Reach DApps and have already completed one or two of our [tutorials](##tuts). 
+
+Complete them all to become a Reach rockstar.
```

Comment:
What is the "them" this is referring to?

Reply:
The tutorials -- edited to make language more clear